brand-guidelines/minicons.md

```diff
@@ -224,8 +224,9 @@ or follows standards of a form-level alert within an
 ### Minicon font
 Our full minicon set is available for desktop use as an icon font (.ttf). It can be accessed through programs like Illustrator and InDesign via the glyph panel.
 
-<a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fcfpb%2Fcf-icons%2Fblob%2Fgh-pages%2Fsrc%2Ffonts%2Fcf-icons.ttf%3Fraw%3Dtrue&sa=D&sntz=1&usg=AFQjCNEfA_Pb9TWATcmS11SAt2epvxOmXg" target="_blank" class="btn">
-    <span class="btn_icon__left cf-icon cf-icon-download"></span>
+<a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fcfpb%2Fcf-icons%2Fblob%2Fgh-pages%2Fsrc%2Ffonts%2Fcf-icons.ttf%3Fraw%3Dtrue&sa=D&sntz=1&usg=AFQjCNEfA_Pb9TWATcmS11SAt2epvxOmXg" target="_blank" class="a-btn">
```

Comment:
What is the deal with this link? Can we fix that?

Reply:
Lol, I have no idea. Fixed.